### ISSUE TITLE
AVT 3 - Carbon X Structured List does not announce checkbox or checkbox status using a screen reader

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#1937]
## Environment
macOS Mojave version 10.14.2
Chrome version 72.0.3626.96 (Official Build) (64-bit)

Windows 7
Chrome version 72.0.3626.96 (Official Build) (64-bit)
JAWS

## Description
AVT 3 - Structured List does not announce that there is a checkbox or the check box state.

1. Navigate to the first row in the list by pressing the tab key.
2. Press Ctrl-Option Right Arrow and navigate across the row.

Actual result: The checkbox is skipped and the status is not announced.
Expected result: VO should announce the checkbox and the state of checked or unchecked.

Note: Add the ARIA role="checkbox" and aria-checked="true"(or false when unchecked).

**Comment by dakahn** [comment]
Great catch! Will need to do a little further research and see if this fails for VO and NVDA as well.

**Comment by snidersd** [comment]
This issue should remain open until it is fixed and retested.

**Comment by snidersd** [comment]
Retested and this is still an issue in Structured List with Selection. 
For React: Tab to selection and keyboard does not allow a selection or announce which one is selected. 
For Vanilla: Tab to selection and hit Enter to make a selection, the check box displays but does not hold the setting or announce that it is checked. 

The keyboard issues are present with and without of screenreader for both Vanilla and React.

**Comment by joshblack** [comment]
Resolution for this is to use the `visually-hidden` class on the `<input>` instead of `display: none` as this removes it from the accessibility tree.

**Comment by emyarod** [comment]
I believe this is blocked similar to #1882 esp if the markup is to be updated. substituting `display: none` with the `visually-hidden` classes introduces new DAP errors which don't seem to be false positives

**Comment by joshblack** [comment]
Following up on this now, our first step was to update the markup to contain the relevant ARIA roles for tabular data so that the control could be navigable.

Marking as blocked as we are unsure how to proceed with implementing selection. In particular, how do we communicate that a row has been selected to a screen reader.

**Comment by dakahn** [comment]
Tested on Windows 10 latest in Firefox and cannot make a selection with the keyboard :hurtrealbad:
